client/app/scripts/superdesk-highlights/highlights.js

```diff
@@ -117,6 +117,40 @@
         };
     }
 
+    MultiMarkHighlightsDropdownDirective.$inject = ['desks', 'highlightsService', 'multi'];
+    function MultiMarkHighlightsDropdownDirective(desks, highlightsService, multi) {
+        return {
+            templateUrl: 'scripts/superdesk-highlights/views/mark_highlights_dropdown_directive.html',
+            link: function(scope) {
+
+                scope.mark_item = function mark_item(highlight) {
+                    angular.forEach(multi.getItems(), function(item) {
+                        if (!item.highlights) {
+                            item.highlights = [highlight._id];
+                        } else if (item.highlights.indexOf(highlight._id) === -1) {
+                            item.highlights = [highlight._id].concat(item.highlights);
+                        } else {
+                            return;
+                        }
+                        highlightsService.mark_item(highlight._id, item._id);
+                    });
+                    multi.reset();
+                };
+
+                scope.is_marked = function is_marked(highlight) {
```

Comment:
Redundant function name? (here and and a few lines above)
Since the function is bound to scope, it is not anonymous anymore...

Also, camelCase should be used for consistency.